src/components/Auth/ChangePassword/ChangePassword.react.js

```diff
@@ -20,165 +20,143 @@ export default class ChangePassword extends Component {
     super(props);
 
     this.state = {
-      passwordError: true,
-      newPasswordError: true,
-      newPasswordConfirmError: true,
-      newPasswordValue: '',
-      passwordValue: '',
-      confirmNewPasswordValue: '',
+      password: '',
+      passwordError: false,
```

Comment:
Only `passwordErrorMessage` state variable is required for the functionality to work.
Same for - 
* `newPasswordErrorMessage`
* `newPasswordConfirmErrorMessage`